src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public Ball getRandomBall() {
+        return new Ball(new Random().nextInt(100), new ColorSupplier().getRandomColor().toString());
```

Comment:
```suggestion
        return new Ball(new Random().nextInt(100), new ColorSupplier().getRandomColor());
```

no need again getting string. You are return string from method getRandomColor()

Reply:
move random and colorsupplier to the class field

Reply:
make 100 as class constant